src/backoff.ts

```diff
@@ -81,7 +90,7 @@ export async function retry(
   options: BackoffOptions,
   targetFunction: any,
   thisArg: any = null,
-  ...args: any[],
+  ...args: any[] // tslint:disable-line
```

Comment:
Look like a bug in tslint: https://github.com/palantir/tslint/issues/4172
According to that issue, another workaround is to add `esSpecCompliant: true` to the `trailing-comma` rule.

Reply:
> Look like a bug in tslint: [palantir/tslint#4172](https://github.com/palantir/tslint/issues/4172)
> According to that issue, another workaround is to add `esSpecCompliant: true` to the `trailing-comma` rule.

@erykwarren hmmm I can't get the option to work, it keeps complaining (and it's supposed to be true by default). Not going to dig more, leaving the `disable-line`, anyway [tslint is getting replaced by eslint](https://medium.com/palantir/tslint-in-2019-1a144c2317a9).